src/ResourceManager/Advisor/Commands.Advisor/Cmdlets/Utilities/SuppressionHelper.cs

```diff
@@ -0,0 +1,71 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Advisor.Cmdlets.Utilities
+{
+    using System;
+    using Microsoft.Azure.Commands.Advisor.Cmdlets.Models;
+
+    public class SuppressionHelper
+    {
+        /// <summary>
+        /// Parse the subscriptionId from the resourceId.
+        /// </summary>
+        /// <param name="resourceID">ResourceId of recommendation</param>
+        /// <returns>SubscriptionId as string</returns>
+        public static string GetSubscriptionIdfromResourceID(string resourceID)
```

Comment:
Please use this: https://github.com/Azure/azure-powershell-common/blob/master/src/ResourceManager/Version2016_09_01/Utilities/Models/ResourceIdentifier.cs

Reply:
We have a custom condition specific to our API. So we will use the existing util.